test/test_performFunction_Calc.c

```diff
@@ -0,0 +1,32 @@
+#ifdef TEST
+
+#include "unity.h"
+
+#include "performFunction_Calc.h"
+
+#include "kcg_consts.h"
+
+
+void setUp(void)
+{
+}
+
+void tearDown(void)
+{
+}
+
+void test_adding_function_adds(void)
+{
+    kcg_float32 val1 = 1;
+    kcg_float32 val2 = 8;
+    kcg_float32 expected = 6;
```

Comment:
Is this intended to fail? @ahouse13

Reply:
Resolving to merge. Can look at results in sonarcloud later

Reply:
Yes it was intended to fail